typescript/lib/parsers/openai.ts

```diff
@@ -536,13 +538,15 @@ export class OpenAIChatModelParser extends ParameterizedModelParser<Chat.ChatCom
       const outputs: ExecuteResult[] = [];
       const responseWithoutChoices = _.omit(response, "choices");
       for (const choice of response.choices) {
+        const messageWithoutContent = _.omit(choice.message, ["content"]);
         const output: ExecuteResult = {
           output_type: "execute_result",
-          data: { ...choice.message },
+          data: choice.message?.content,
           execution_count: choice.index,
           metadata: {
             finish_reason: choice.finish_reason,
             ...responseWithoutChoices,
+            ...messageWithoutContent,
```

Comment:
I'm concerned the metadata won't make much sense if message is saved without content here. I'm guessing this would just be "role" then? Need to think about it some more but fine for now

Reply:
It may make sense to just have a `message`, `rawMessage` or `rawResponse` property in metadata that stores the entire raw response.